SYCL/Reduction/reduction_nd_N_queue_shortcut.cpp

```diff
@@ -0,0 +1,144 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+
+// `Group algorithms are not supported on host device.` on NVidia.
+// XFAIL: hip_nvidia
```

Comment:
I wonder if this is still applicable. Should we rerun the tests?

Reply:
I think normal CI would report/complain on this test if it passed on nvidia, right?

Reply:
If it is the case for other similar tests, then I suspect you are right. If it starts unexpectedly passing we'll know to remove it. 😄